spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -1652,4 +1652,16 @@ An `InterceptionFactory` can be obtain be calling `BeanManager.createInterceptio
 
 The container must provide a bean with scope `@Dependent`, bean type `InterceptionFactory` and qualifier `@Default` allowing producer methods to obtain this factory to produce their instance.
 
+Following example demonstrates producer method definition using `InterceptionFactory`:
+
```

Comment:
you could perhaps add a few word to say that the returned value will have the interceptor binding `ActionBinding`